server/model/taggedBailReviewModel.ts

```diff
@@ -23,8 +28,12 @@ export default class TaggedBailReviewModel {
       ),
     )
 
-    return `${selectedCase.courtDescription}<br>${selectedCase.caseReference || ''} ${dateToString(
-      new Date(selectedCase.sentenceDate),
-    )}`
+    return isSentenceRecalled(selectedCase.sentenceCalculationType)
+      ? `${selectedCase.courtDescription} ${getSentenceRecallTagHTML()}<br>${selectedCase.caseReference || ''} ${dateToString(
```

Comment:
And here.

Reply:
I've updated the code as you've suggested above